src/query/storages/system/src/query_log_table.rs

```diff
@@ -367,6 +376,27 @@ impl SystemLogElement for QueryLogElement {
             .next()
             .unwrap()
             .push(Scalar::String(self.query_text.clone()).as_ref());
+        let query_hash = format!("{:x}", Md5::digest(&self.query_text));
```

Comment:
We can add `query_hash` and `query_parameterized_hash` to `QueryContext`, it can be easier to reuse, and at the same time we can use the Statement that have already been parsed and don't need to re-parse them again.

Reply:
Done